articles/healthcare-apis/dicom/dicom-services-conformance-statement-v2.md

```diff
@@ -610,6 +612,7 @@ The query API returns one of the following status codes in the response:
 | `400 (Bad Request)`         | The server was unable to perform the query because the query component was invalid. Response body contains details of the failure. |
 | `401 (Unauthorized)`        | The client isn't authenticated. |
 | `403 (Forbidden)`              | The user isn't authorized. |
+| `424 (Failed Dependency)`      | The DICOM service cannot access a resource it depends on to complete this request. An example is failure to access the connected Data Lake store, or the key vault for supporting customer-managed key encryption. |
```

Comment:
Talked offline - 

Creating a generic message without being specific of the use cases is tricky. I'm not sure if customers would understand messaging like "resources managed by you" because they see things like the connected store & CMK as "Azure" still. 

If we can think of a good generic message, we can update all of these descriptions to be the same. @mmitrik any ideas here?

Otherwise I'll just remove the connected store example from qido and leave the rest as-is.